### DISCUSSION TITLE
Add Mockaroo integration

### DISCUSSION THREAD

**Discussion by X3Technology** [DNNCommunity/Dnn.StructuredContent#101] [Ideas]
Integrate with Mockeroo https://mockaroo.com/ to provide sample test data.

User will have to provide a API key (there is a free option).
System will call Mockeroo API with parameters of the current ContentType (field names, data types, etc.) and then populate the table with the returned mock data.

**Comment by mitchelsellers** [comment]
Personally, at this stage of the game, I would suggest not integrating with third parties.  Every time we integrate with a third party we introduce a dependency that can change and if it does change and we cannot react quick enough the system will fail.  

Once more mature as a product I think it could be great, but a bit early in my opinion.
